### DISCUSSION TITLE
[zh_CN] General Translations Fix

### DISCUSSION THREAD

**Discussion by wensimehrp** [OpenTTD/team#472] [Other]
@OpenTTD/zh_cn 

*As this is a problem related to languages, Chinese (Mandarin) will be used in this discussion.*

## 问题

当前版本的OpenTTD中还存在一些不适当的翻译：
- 译文中机翻过多
- 部分译文使用语句冗杂/不通顺
- 对于原文当中使用同一单词的，有多个对应的译文用词，如Train -> 列车/火车
- 译文与原文不符
- 等等

## 目标

- 更正翻译

## 措施

- 校对目前所有译文
- 对于原文中出现的高频词汇，建立一个对应的列表，统一译文用词

由于eints本身的限制，逐一修改译文非常麻烦。因此目前计划如果提议有人响应，就建立一个仓库来储存翻译，最后建立PR。

*以下一些个人感想*

我个人认为，翻译的首要目标是让人能看懂，为了“能看懂”这个目标，对原文进行适当修改是可以忍受的，前提是没有其他的译文是建立在这项译文之上的。另外，一些项目是没有必要翻译的，如NewGRF开发中的部分术语，翻译过后反而难以理解。

**Comment by clzls** [comment]
我认为建立“术语表”（terminology）有助于改善上述大部分翻译问题，不过现在的问题可能是人手不足，以及需要与开发组协商如何优化翻译流程。我相信术语混乱不是中文一家的问题。

I think making a list of _terminology_ is very helpful when dealing with these problems. But we don't have enough hands on it and how to improve the translate procedure together with the dev group. I think problems about terminology is not unique to Chinese.

**Comment by wensimehrp** [comment]
这是一个样例

<img width="171" alt="image" src="https://github.com/OpenTTD/team/assets/145064583/b9adb707-729b-4959-8ed0-ac9b20c94f46">


<img width="298" alt="image" src="https://github.com/OpenTTD/team/assets/145064583/23c54159-cbca-44ca-8757-26fb7d927e04">


<img width="372" alt="image" src="https://github.com/OpenTTD/team/assets/145064583/eae0022a-cd47-41d8-ac9c-58011b00be6a">


对比三张图片，可以至少发现两个问题：
一是第一张图片中的“非粘滞”，本身原文的意思是“除了已经固定的窗口”，而“粘滞”这个词较少使用（至少是对我个人而言），出现在此项中难以让人理解。
二是一、三图中对于“On”有多种翻译（不过图一原文是"Yes"与"No"）包括“打开”、“启用”与“是”。我建议统一修改成“启用”，以保持样式的统一。

另外，原文中的Road Vehicles被翻译成"汽车”，完全忽略了游戏中的另一种载具即轨道车，或者说电车。我建议修改成“道路载具”或者“公路载具”，同理，“飞机”修改成“航空器”也更加符合原意。

除此之外，原文在设置中有出现Legacy一词，被翻译成"传统”，但"传统"一词（在我个人看来）更包含一种褒义，而Legacy本身指“遗产，遗留物”，在原文中的意思更多是"可以使用，但是不建议使用"。翻译成"传统"（还是在我个人看来）可能会鼓励玩家去使用过时的设置，如修改调色盘等。

对于人手不足的问题，对应的措施只有加大宣传。GitHub平台本身在国内的特殊处境也是一个问题。

**Comment by wensimehrp** [comment]
<img width="339" alt="image" src="https://github.com/OpenTTD/team/assets/145064583/abc693e1-a883-4376-98f5-d8c4d9c5dc6c">

此处为错译之范例。什么是“加强版固定资产维护“？"加强”在哪里？

**Reply by wensimehrp** [reply]
<img width="627" alt="image" src="https://github.com/OpenTTD/team/assets/145064583/fe6e329d-92b4-4d73-b5b4-e7747589312e">

对应的说明也是错误的

**Comment by wensimehrp** [comment]
<img width="218" alt="image" src="https://github.com/OpenTTD/team/assets/145064583/b3c0f70a-8429-47b2-a79e-e27a2bd0297b">

一个另外的问题是，设置项本身有“启用”，但是后面又接上了“打开”，等于对同一事物“启用”了两次。

**Comment by wensimehrp** [comment]
现在的翻译文本中掺杂中文标点和英文标点，虽然在外观上没有明显的区别，但是我建议全部修改为中文标点。

**Comment by wensimehrp** [comment]
## 关于游戏中的术语

游戏中存在相当多的术语，用来指代一些特殊的含义。术语表中也包含一些对某些条目的具体修改。

### 城镇与世界生成

| 原文 | 译文 | 目前译文 | 备注 |
| --- | --- | --- | --- |
|temperate|温带|温带|
|sub-arctic|亚寒带|寒带|翻译为“寒带”可能更好？|
|sub-tropical|亚热带|沙漠|翻译为“热带”可能更好？|
|toyland|奇幻|玩具、奇幻|出现一词多译|
| --- | --- | --- | --- |
|Landscape|景观|景观||
|Land generator|地形算法|地形生成器|目前译文来自13.4|
|Original|原始算法|原始算法|地形生成器的子条目|
|TerraGenesis|TerreGenesis|新算法|目前译文来自13.4|
|Terrain type|地形|地貌类型||
|density|密度|布局|如Town density、industry density|
|Snow/Desert coverage|雪地/沙漠比率|积雪/沙漠覆盖率|目前译文来自13.4|
|Roughness of terrain|地形表面|地面粗糙度||
|Variety distribution|地形变化|多样的分发||
|Tree placer algorithm|森林算法|森林算法||
|Original|原版|原始的|森林算法的子条目|
|Improved|改良|增强的|森林算法的子条目|
|Road vehicles|车辆通行方向|道路车辆|本项用于指定道路载具的通行方向，此项我认为可以不用“道路载具通行方向”而使用更常见的“车辆”|
|town|城镇|城镇、市镇、地区、城市||
|city/cities|都市|城市、城镇|city是一种特殊的town，具有更高的初始人口和更快的发展速度。在中文语境中，“城市”相较于“都市”被更频繁地使用，因此我建议使用“都市”来减少可能出现的指代的错误|

### 基础

|原文|译文|目前译文|备注|
|---|---|---|---|
| Multiplier | 系数 | 因子、倍数、系数 |
| Readme | 说明 | 读我、说明、readme| readme出现于“……请参阅readme.txt……”中，不需要修改|
| On/Enable |启用|打开、允许、启用||
|Off/Disable|禁用|禁用、关闭||

### 基础建设

|原文|译文|目前译文|备注|
|---|---|---|---|
|Railway track/Railway|铁轨|铁轨、铁路轨道||
|Road|道路|道路、公路||
|Tramway|轨道车轨道|电车道|译文可能有些不妥|

### 未归类
|原文|译文|目前译文|备注|
|---|---|---|---|
|Original|原版|原版||
|Legacy|旧版|传统、原版||
|Default|默认|默认、缺省||

**Comment by wensimehrp** [comment]
目前我已经修正了一部分翻译。主要错误已经修正，仍有小部分需要调整。
